readme.md

```diff
@@ -159,7 +164,7 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
 * **Your Answer:** 
-
+the delete method - you can return the deleted item
```

Comment:
You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.